frontend/src/components/InfoOrNull.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+    <slot v-if="condition"></slot>
```

Comment:
eslint seems to be throwing an error on this line:

![image](https://user-images.githubusercontent.com/7469472/128210445-46e7f019-ba75-4ef3-9e9c-52e8e1b41427.png)

is this something to be concerned of?

Reply:
As embarrassing as it might be, from a guy that has been working on a static analyzer for years, I didn't use a linter. From what I understood, the rule was created for Vue v1 and 2 because by design it didn't allow templates to have text as root, multiple elements or even certain directives on its root element. The documentation doesn't say why and honestly I can't imagine why it is so. On Vue 3 however, it seems like this no longer applicable. I suppose the linter rule is not checking for the framework version yet.